Packet++/src/X509Decoder.cpp

```diff
@@ -975,7 +975,7 @@ namespace pcpp
 	std::unique_ptr<X509Certificate> X509Certificate::fromPEM(const std::string& pemData)
 	{
 		auto derData = PemCodec::decode(pemData, certificatePemLabel);
-		std::unique_ptr<uint8_t[]> derDataBuffer(new uint8_t[derData.size()]);
+		auto derDataBuffer = std::make_unique<uint8_t[]>(derData.size());
```

Comment:
This change addresses a previous PR comment: https://github.com/seladb/PcapPlusPlus/pull/1913#discussion_r2256546073